Makefile

```diff
@@ -20,6 +20,7 @@ endif
 
 init::
 	python -m pip install pip-tools
+	python -m pip install gitpython
```

Comment:
So for some reason this package doesn't seem to be adding to or loading from the requirements file when using piptools?? is this ok being added here or can anyone advise on why this might not be working?